apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -5,15 +5,22 @@ module.exports = {
     "airbnb",
     "prettier",
```

Comment:
Should we update prettier to use the same proper config ? https://github.com/prettier/eslint-plugin-prettier#recommended-configuration
```suggestion
    "plugin:prettier/recommended",
```